src/file_manager.h

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2015 Alexander Zolotarev <me@alex.bio> from Minsk, Belarus
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*******************************************************************************/
+
+#ifndef FILE_MANAGER_HPP
+#define FILE_MANAGER_HPP
+
+#include <string>
+#include <functional>
+#include <fstream>
+
+namespace alohalytics {
+
+// Functions are wrapped into the class for convenience.
+struct FileManager {
+  // Initialized separately for each platform.
+  static const char kDirectorySeparator;
+
+  // Checks and appends if necessary platform-dependent slash at the end of the
```

Comment:
Methods are more readable in header-only concept if divided by new lines, so it is intentional.

> On Jun 10, 2015, at 20:00, Dima <notifications@github.com> wrote:
> 
> In src/file_manager.h <https://github.com/KnowSheet/Alohalytics/pull/36#discussion_r32140008>:
> 
> > +
> > +#ifndef FILE_MANAGER_HPP
> > +#define FILE_MANAGER_HPP
> > +
> > +#include <string>
> > +#include <functional>
> > +#include <fstream>
> > +
> > +namespace alohalytics {
> > +
> > +// Functions are wrapped into the class for convenience.
> > +struct FileManager {
> > +  // Initialized separately for each platform.
> > +  static const char kDirectorySeparator;
> > +
> > +  // Checks and appends if necessary platform-dependent slash at the end of the
> Unnecessary line break.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KnowSheet/Alohalytics/pull/36/files#r32140008>.
>